// @expo-google-fonts/urbanist/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Urbanist_100Thin = require('./100Thin/Urbanist_100Thin.ttf');
export const Urbanist_200ExtraLight = require('./200ExtraLight/Urbanist_200ExtraLight.ttf');
export const Urbanist_300Light = require('./300Light/Urbanist_300Light.ttf');
export const Urbanist_400Regular = require('./400Regular/Urbanist_400Regular.ttf');
export const Urbanist_500Medium = require('./500Medium/Urbanist_500Medium.ttf');
export const Urbanist_600SemiBold = require('./600SemiBold/Urbanist_600SemiBold.ttf');
export const Urbanist_700Bold = require('./700Bold/Urbanist_700Bold.ttf');
export const Urbanist_800ExtraBold = require('./800ExtraBold/Urbanist_800ExtraBold.ttf');
export const Urbanist_900Black = require('./900Black/Urbanist_900Black.ttf');
export const Urbanist_100Thin_Italic = require('./100Thin_Italic/Urbanist_100Thin_Italic.ttf');
export const Urbanist_200ExtraLight_Italic = require('./200ExtraLight_Italic/Urbanist_200ExtraLight_Italic.ttf');
export const Urbanist_300Light_Italic = require('./300Light_Italic/Urbanist_300Light_Italic.ttf');
export const Urbanist_400Regular_Italic = require('./400Regular_Italic/Urbanist_400Regular_Italic.ttf');
export const Urbanist_500Medium_Italic = require('./500Medium_Italic/Urbanist_500Medium_Italic.ttf');
export const Urbanist_600SemiBold_Italic = require('./600SemiBold_Italic/Urbanist_600SemiBold_Italic.ttf');
export const Urbanist_700Bold_Italic = require('./700Bold_Italic/Urbanist_700Bold_Italic.ttf');
export const Urbanist_800ExtraBold_Italic = require('./800ExtraBold_Italic/Urbanist_800ExtraBold_Italic.ttf');
export const Urbanist_900Black_Italic = require('./900Black_Italic/Urbanist_900Black_Italic.ttf');
